packages/web/src/components/organisms/MarketOverview/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Row, Col } from 'antd'
+import Link from 'next/link'
+import { useGetMarketFactoryCreateQuery } from '@dev/graphql'
+import { useGetMarketInformation } from 'src/fixtures/github/hooks'
+
+const Text: React.FC = ({ children }) => (
+  <div
+    style={{
+      fontSize: '18px',
+      lineHeight: '24px',
```

Comment:
We may be able to remove implementation of specific font size like this 🙆‍♀️

Reply:
I'm in favor of unifying or reducing font size specifications!